hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/security/TestRouterSecurityManager.java

```diff
@@ -39,6 +39,7 @@
 import org.junit.Rule;
 import org.junit.Test;
 
+import static org.apache.hadoop.metrics2.util.Metrics2Util.*;
```

Comment:
just suggest to replace with single class imports.

Reply:
Sure